common/djangoapps/terrain/ui_helpers.py

```diff
@@ -537,6 +537,15 @@ def click_link(partial_text, index=0):
 
 
 @world.absorb
+def click_button(data_attr, index=0):
+    xpath = '//button[text()="{button_text}"]'.format(
+        button_text=data_attr
+    )
+    world.browser.find_by_xpath(xpath)[index].click()
+    wait_for_js_to_load()
```

Comment:
I copied the example above (a link, I think?) and changed it to look for a button. This `wait_for_js_to_load()` I assumed was necessary.